.ci/jobs/build-docker-images.yml

```diff
@@ -17,6 +22,6 @@
             credentials-id: f6c7695a-671e-4f4f-a331-acdce44ff9ba
             reference-repo: /var/lib/jenkins/.git-references/e2e-testing.git
             branches:
-              - master
+              - $branch_specifier
```

Comment:
Do you need to change -> https://github.com/elastic/e2e-testing/blob/master/.ci/build-docker-images.groovy#L35?

OTOH, I don't think you need this parameters but https://github.com/elastic/apm-pipeline-library/blob/64546ee45b1c64b11d6138053401062d51c7e36f/resources/JenkinsfileTemplate.groovy#L160-L163

In other words, if the issue is related to the first build, then you need to set the parameters with a default value and create and env variable. This is the workaround with the 🐔 🥚 

-

Reply:
You mean removing the parameter from JJBB and adding the parameter block in the groovy file?

Reply:
yes